src/AutoPilotPlugins/PX4/PX4TuningComponentPlaneRate.qml

```diff
@@ -0,0 +1,173 @@
+                    step:           0.0002
+                }
+                ListElement {
+                    title:          qsTr("Integral Gain (${param})")
+                    description:    qsTr("Generally does not need much adjustment, reduce this when seeing slow oscillations.")
+                    param:          "FW_PR_I"
+                    min:            0.0
+                    max:            0.5
+                    step:           0.025
+                }
+                ListElement {
+                    title:          qsTr("Feedforward Gain (${param})")
+                    description:    qsTr("Feedforward gused to compensate for aerodynamic damping.")
+                    param:          "FW_PR_FF"
+                    min:            0.0
+                    max:            10.0
+                    step:           0.05
+                }
+            }
+        }
+        property var yaw: QtObject {
+            property string name: qsTr("Yaw")
+            property var plot: [
+                { name: "Response", value: globals.activeVehicle.yawRate.value },
+                { name: "Setpoint", value: globals.activeVehicle.setpoint.yawRate.value }
+            ]
+            property var params: ListModel {
+                ListElement {
+                    title:          qsTr("Porportional Gain (${param})")
+                    description:    qsTr("Porportional Gain.")
+                    param:          "FW_YR_P"
+                    min:            0.0
+                    max:            3
+                    step:           0.05
+                }
+                ListElement {
+                    title:          qsTr("Integral Gain (${param})")
+                    description:    qsTr("Generally does not need much adjustment, reduce this when seeing slow oscillations.")
+                    param:          "FW_YR_D"
+                    min:            0.0
+                    max:            0.4
+                    step:           0.02
+                }
+                ListElement {
+                    title:          qsTr("Integral Gain (${param})")
+                    description:    qsTr("Generally does not need much adjustment, reduce this when seeing slow oscillations.")
+                    param:          "FW_YR_I"
+                    min:            0.0
+                    max:            0.5
+                    step:           0.025
+                }
+                ListElement {
+                    title:          qsTr("Feedforward Gain (${param})")
+                    description:    qsTr("Feedforward gused to compensate for aerodynamic damping.")
+                    param:          "FW_YR_FF"
+                    min:            0.0
+                    max:            10.0
+                    step:           0.05
+                }
+                ListElement {
+                    title:          qsTr("Roll control to yaw feedforward (${param})")
```

Comment:
That only works if you use template strings:
https://www.w3schools.com/js/js_string_templates.asp
[Example](https://qmlonline.kde.org/?code=JYWwDg9gTgLgBARRggrsAxgazgJgHQDsAUKJLIsmlngMIQB2MUEANgM654DMRRASgFN0MAIb0A5iwFwA3kTgK46VtABccAERQBAEw3zFY9AAtobPADNgLFurAjtjXorgAVAQA94cly5ieYdQADABIZehEQAQBfIINfODBmMAFYAE84NiZgCTgIqPUNAHUAUTLyiv0EhQsGGDwwYA8BFgBlYAAvAXUAVgAGeN9lFjVNAHdjYH8q6qNTKHN0AUZUgEl6OwdlmEGXPghRGGAGAEF6UBEYaDgGOGZD4-pZXeqoFHpziXUmFAEXhJGEDAbHUZwuRwYeHWVnO-n+vgszBA6gG1V8V3UXAAbKi0YodCgoJdHuoAIz9XEJaKDanUoA)

Reply:
I am not so familiar with qmls, but I think this is not done for other tuning components

Reply:
> I am not so familiar with qmls, but I think this is not done for other tuning components - how can I use template strings in a qml?

Check both links that I put in my previous comment, there is the js documentation and also a qml example using qmlonline

Reply:
@patrickelectric I don't think I really understand what needs to be done

Reply:
You need to use the "string template", to do that, use ``` ` ``` over ``` " ```.
Check the qml example.

Reply:
From the doc:
![image](https://user-images.githubusercontent.com/1215497/188134214-5772bbb1-4dff-4efc-b432-7c879809ae5a.png)

Reply:
This doesn't work: 
```
                ListElement {
                    title:          qsTr('Differential Gain (${param})')
                    description:    qsTr("Damping: increase to reduce overshoots and oscillations, but not higher than really needed.")
                    param:          'FW_PR_D'
                    min:            0.0
                    max:            1.00
                    step:           0.0002
                }
```

Reply:
Please, read my previous comment and also the documentation. You MUST use back-ticks over quotes.

Reply:
I created a commit over your PR, please check: https://github.com/mavlink/qgroundcontrol/pull/10384/commits/474c42ca7ac67b1024059e708eb40bb385753f0e